OG/OG/ASTBuilding/TreeNodes/BodyNode and Statements/Statements/AssignmentNodes and extractors/AssignmentNodeExtractor.cs

```diff
@@ -169,7 +169,7 @@ public IdAssignNode ExtractAssignmentNode(OGParser.IdAssignContext context)
         {
             string id = context.id.Text;
             string value = context.value.Text;
-            Console.Write("\t\tCreating IDAssignmentNode from expression {0} = {1}.", id, value);
+            //Console.Write("\t\tCreating IDAssignmentNode from expression {0} = {1}.", id, value);
```

Comment:
Puha... jeg ser altså helst de bliver slettet ;-)
udkommenteret kode er ufatteligt forvirrende at læse, synes jeg.
Hvis det tager for lang tid at ændre, så kan jeg se gennem fingre med det :-P